src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAYMENT = 40;
+  const BIGGER_DISCONT = 7;
+  const SMALLER_DISCONT = 3;
```

Comment:
Similarly, 'SMALLER_DISCONT' should be renamed to be more descriptive. A better name could be 'SHORT_TERM_RENTAL_DAYS'.